<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
<html>
<head>
   <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
   <meta name="Description" content="Cell and Xenopus methods ala Klymkowsky et al">
   <meta name="description" content="MCD Biology 1150">
   <meta name="keywords" content="Cell biology, Evolution, Disease">
   <meta name="GENERATOR" content="Mozilla/4.7 (Macintosh; I; PPC) [Netscape]">
   <meta name="KeyWords" content="Cell Biology, Xenopus">
   <meta name="Content-Type" content="text/html; charset=iso-8859-1">
   <title>Klymkowsky Lab Methods - Plasmid Construction Methods</title>
</head><style type="text/css">
   <!--
   a:link {  text-decoration: none}
   a:visited {  text-decoration: none}
   a:active {  text-decoration: none}
   a:hover {  text-decoration: bold}
   -->
   </style>
<body text="#000000" bgcolor="#FFFFFF" link="#000099" vlink="#990000" alink="#990066">
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="423"><div align="left"> 
        <p><font color="#000000" size="+1" face="Book Antiqua"><i><strong>Plasmid construction </strong></i></font></p>
        </div>
      <div align="right"> 
        <table width="100%" border="0" cellpadding="0" cellspacing="0">
          <tr> 
            <td width="53%"><font size="-1" face="Arial, Helvetica, sans-serif"><em><strong><a href="../methods.htm" target="_self"><img src="../Graphics/ArrowLEFT.GIF" width="12" height="12" border="0"> Table of Methods</a></strong></em></font></td>
            <td width="47%"><div align="right"><font size="-1" face="Book Antiqua"><em>6 
              December 2002</em></font> <em>latest update 24 july 2019</em> </div></td>
          </tr>
        </table>
        <p><font size="-1" face="Arial, Helvetica, sans-serif">PCR | Blunting 
          | Ligation | Transformation</font></p>
      </div> </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr>
    <td>
      <hr>
    </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr>
    <td><p><font face="Arial, Helvetica, sans-serif"><strong><font color="#990000">Polymerase 
      chain reaction: </font></strong>&nbsp;</font></p>
      <p><strong><font face="Arial, Helvetica, sans-serif">here is a good on-line reference for <a href="http://cafgroup.lbl.gov/protocols/general-molecular-biology/pcr-amplification-of-genomic-or-plasmid-dna" target="_blank">PCR amplification</a></font></strong></p>
      <p><font face="Arial, Helvetica, sans-serif">100 &micro;L reaction (two 50&micro;L tubes) </font><font face="Arial, Helvetica, sans-serif">- for subcloning - for PCR verification you need only 50 &micro;L reaction.</font></p>
      <p><font face="Arial, Helvetica, sans-serif">10 &micro;L 10XThermoPol buffer<br>
      1.6  &micro;L mixed dNTPS (each at 25 mM)<br>
      </font><font face="Arial, Helvetica, sans-serif">1  &micro;L each forward and backward primers (from 100 &micro;L stocks)<br>
      </font><font face="Arial, Helvetica, sans-serif">1  &micro;L plasmid<br>
      </font><font face="Arial, Helvetica, sans-serif">1 &micro;L</font><font face="Arial, Helvetica, sans-serif">Vent enzyme waterto 100 &micro;L, 50 &micro;L per tube</font></p>
      <p><font face="Arial, Helvetica, sans-serif"><strong>Run program Vent1</strong> <br>
        - adjust annealing time 5&ordm;C belong lowest primer Tm<br>
      - extension time 1 min per 1000 base pair lenght.<br>
      - 30 cycles / hold overnight at 4&ordm;C - take in the morning <br>
      </font><font face="Arial, Helvetica, sans-serif">- do not leave machine running more than 18hours. </font></p>
      <p><font face="Arial, Helvetica, sans-serif"><strong>You can </strong><br>
      check PCR by running 2.5-5  &micro;L on gel<br>
      </font><font face="Arial, Helvetica, sans-serif">cut the PCR using CutSmart buffer - cut 3 to 5  &micro;L of PCR reaction (should be plenty) in 30 to 40 uL reaction<br>
        <strong><br>
        This is my old  set up for simple amplification from a plasmid 1 to 5 &micro;L 
        of plasmid (miniprep) DNA</strong><br>
        30 &micro;L nucleotides<br>
        20 &micro;L polymerase buffer <br>
        2 &micro;L of each primer (1 &micro;g/&micro;L)<br>
        1 &micro;L MgSO4 stock (100mM)<br>
        2 &micro;L Vent <br>
        <br>
        water to 200 &micro;L - run as 50 &micro;L samples (4 tubes) <br>
    </font></p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr>
    <td><hr></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td> <p><font color="#990000" face="Arial, Helvetica, sans-serif"><strong>Checking 
        your plasmids &amp; restriction digests </strong></font></p>
      <p><font face="Arial, Helvetica, sans-serif"> Simple diagnostic digests 
        to determine if you are dealing with the correct plasmid. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">Since this was originally written NEB has introduced CutSmart buffer - which often is the best choice for single and double digests (check the back of the NEB catalog).. </font>&nbsp;</p>
      <p><font face="Arial, Helvetica, sans-serif">Taking DNA from a standard 
        miniprep,<br>
        you would normally analzye 2&micro;L of uncut DNA and 4 to 5&micro;L of a restriction 
        digest. </font></p>
      <p><font face="Arial, Helvetica, sans-serif"><strong>Standard digest </strong></font></p>
      <ul>
        <li><font face="Arial, Helvetica, sans-serif">3 to 5 &micro;L 
          of plasmid DNA </font></li>
        <li><font face="Arial, Helvetica, sans-serif"> 3 &micro;L of appropriate 
          buffer. Each enzyme has its own <strong>optimal buffer</strong> -- check 
          chart to determine which is best for the enzyme(s) you are using.</font>&nbsp; </li>
        <li><font face="Arial, Helvetica, sans-serif">1 &micro;L of each enzyme 
          (<font color="#FF0000"><strong>never</strong></font> more than 10% enzyme!)</font></li>
        <li><font face="Arial, Helvetica, sans-serif"> water to 30 &micro;L </font></li>
      </ul>
      <p><font face="Arial, Helvetica, sans-serif">Always check that you know 
        the temperature at which the enzymes cuts!</font></p>
      <p><font face="Arial, Helvetica, sans-serif">Standard conditions, 1 hour 
        @ 37&deg;C. </font></p>
      <p><font face="Arial, Helvetica, sans-serif"><br>
        ALWAYS <font color="#990000"><strong>KEEP ENZYMES IN THE BIG PIG</strong></font>, 
        return to -20&deg;C as soon as you are done!<br>
        <br>
        <font color="#990000"><strong>CHANGE PIPETTE TIPS TO AVOID CROSS-CONTAMINATING 
        </strong></font>ENZYMES</font><font face="Arial, Helvetica, sans-serif"><br>
        </font></p>
      </td>
  </tr>
</table>
<table width="720
" border="0" cellpadding="0">
  <tr>
    <td><hr size="1"></td>
  </tr>
</table>
<table width="720
" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td><table width="720
" border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td><p><font face="Arial, Helvetica, sans-serif"><strong>DNA gel analysis:</strong> <br>
          <br>
          </font><font face="Arial, Helvetica, sans-serif">For a large gel: use 0.6-0.8 
            g agarose melted into 100ml TAE buffer<br>
            add 10&micro;L 1mg/ml Ethidium bromide solution</font></p>
          <p><font face="Arial, Helvetica, sans-serif">Dissolve in microwave - 85-90 
            seconds</font></p>
          <p><font face="Arial, Helvetica, sans-serif"> COOL solution to below 60&deg;C 
            before pouring gel <br>
            <br>
            Run gel at betwee 70-100V.<br>
            </font><font face="Arial, Helvetica, sans-serif"><br>
              <strong>NOTE</strong>: Gels can be stored in the cold room for a day or 
              two, so if you pour a large gel, it can be used to analyze a number of 
              samples.</font></p></td>
      </tr>
      <tr>
        <td><hr size="1" noshade>
          <p><font face="Arial, Helvetica, sans-serif"><strong>Isolation of DNA from  gel bands:</strong> <br>
            <br>
            You can cut out gel band using a 22mm coverslip (less dangerous and cheaper than a scalpel)<br>
            then use Qiagen gel band extraction kit: <a href="QIAquick-Gel-Extraction-Kit-EN.pdf" target="_blank"><strong>instructions</strong></a></font></p></td>
      </tr>
    </table></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td> <p><font face="Arial, Helvetica, sans-serif"><strong>Partial digests</strong>: 
        </font></p>
      <p><font face="Arial, Helvetica, sans-serif">In some cases you may need 
        to do a partial digest. This can be tricky. <br>
        <br>
        Always carry out a complete digest to determine the final pattern. <br>
        <br>
        A good starting point is to use the enzyme (1&micro;L) diluted 1:10 in 
        its appropriate buffer + acetylated BSA) and reduce the time and temperature 
        of the incubation. </font></p>
      <p><font face="Arial, Helvetica, sans-serif"><br>
        I suggest 5-10 min. at r.t. </font><font face="Arial, Helvetica, sans-serif"><br>
        </font></p>
      </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td> <hr size="1"> </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td> <p><font face="Arial, Helvetica, sans-serif"><font color="#990000"><strong>Removing 
        a restriction site with 5' overhang</strong></font>: </font><font face="Arial, Helvetica, sans-serif"><strong>Klenow 
        fill-inreaction</strong>: </font></p>
      <p><font face="Arial, Helvetica, sans-serif"> Following restriction reaction 
        add </font></p>
      <p><font face="Arial, Helvetica, sans-serif">5uL of 2.5 mM dNTPs &amp; 5 
        &micro;L of Klenow enzyme. <br>
        Incubate at room temp for 20 minutes and then for 5 minutes at 37&deg;C.<br>
        ligate and transform. </font></p>
      </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td> <hr size="1"> </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td> <font face="Arial, Helvetica, sans-serif"><strong><font color="#990000">Removing 
      a restriction site with a 3' overhang</font></strong>: T4 polymerase &quot;chew 
      back&quot; : </font></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td> <hr size="1"> </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td> <hr size="1"> </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td> <font face="Arial, Helvetica, sans-serif"><strong>Cloning from RT-PCR</strong> 
      (Shana Fawcett) </font></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td> <hr size="1"> </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td> <p><font face="Arial, Helvetica, sans-serif"><strong>Ligation reaction</strong>: 
        </font></p>
      <p><font face="Arial, Helvetica, sans-serif">Take</font><font face="Arial, Helvetica, sans-serif"> 
        DNA (plasmid + insert or whatever) isolated from gel bands </font></p>
      <p><font face="Arial, Helvetica, sans-serif">resuspend in 25 </font><font face="Arial, Helvetica, sans-serif"> 
        &micro;L water (27 uL to elute Quiagen column, 2 &micro;L will be lost) 
        </font></p>
      <p><font face="Arial, Helvetica, sans-serif">add 3 &micro;L T4 ligase buffer 
        and 2&micro;L T4 DNA ligase</font></p>
      <p><font face="Arial, Helvetica, sans-serif"> incubate &gt; 2h at 16&deg;C 
        and then transform into competant bacteria. </font></p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td> <hr size="1"> </td>
  </tr>
</table>
<table width="720" border="0">
  <tr>
    <td><p><font face="Arial, Helvetica, sans-serif"><strong>Standard transformation</strong>:<br>
        1. add 1&micro;L of miniprep DNA or 15&micro;L of a ligation reaction to a vial of 
        competant cells<br>
        2. let sit on ice for 30-180 minutes.<br>
        3. transfer to a plastic-capped tube / heat shock at 42&deg;C for 1 min 
        or 37C for 5 min.<br>
        4. add 0.8 ml LB media - let grow for more than 20 minutues at 37&deg;C<br>
        5. plate out onto LB + AMP plates<br>
    </font><font face="Arial, Helvetica, sans-serif"><br>
        typically for a simple transformation of cells with intact plasmid, I 
        would make two plates, using 50uL and the other with 200uL of cells. <br>
        <br>
        for a ligation reaction, I would plate the entire transformation onto 
        two plates. <br>
        <br>
        Always flame the glass spreader and allow it to cool before using it to 
        spread cells out on a plate. If it is too hot, it will kill the cells. <br>
        <br>
        If you do not flame it, you will cross contaminate cultures (a bad thing 
        to do!). </font></p>
      </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td> <hr size="1"> </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td> <font face="Arial, Helvetica, sans-serif"><strong>Growing up colonies:</strong><br>
      Use a yellow pipette tip and spear a colony. <br>
      Only take colonies that are well seperated from others on the plate. <br>
      Eject the tip into a sterile / capped glass tube<br>
      Add 5ml LB broth and 100mL of ampicillin* (100mg/ml stock)<br>
      Incubate in tube roller (third floor warm room) overnight. <br>
      * Note: if you yeild of plasmid DNA is poor, the most likely problem is 
      that your ampicillin has &quot;gone off&quot;. Make new ampicillin stock 
      and stay aware of this possibility.<br>
      AMPICILLIN STOCK: 100mg/ml in distilled water</font> </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td> <hr size="1"> </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td> <p><font face="Arial, Helvetica, sans-serif"><strong>GeneJet&nbsp; protocol for minipreps&nbsp;</strong><br>
      <br>
1. Pellet 1.5ml of bacteria from an overnighter (3 min. spin in minifuge).<br>
2. Discard supernatants / and add another 1.5 ml of culture - spin down and discard supernatant<br>
3. 
    </font><font face="Arial, Helvetica, sans-serif">resuspend in <strong>250 mL</strong> resuspension buffer (use Alberto's trick)<br>
        3. add 250 &micro;Lcell lysis buffer - invert until clear.<br>
        <br>
    </font><font face="Arial, Helvetica, sans-serif">4. add 350 &micro;L neutralization buffer - invert to mix and then spin in microfuge for 
        5 minutes. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">5. take supernatant - transfer to a GeneJet column (tube) tube <br>
        6. centrifuge 1 min.&nbsp; - discard flow through <br>
      </font><font face="Arial, Helvetica, sans-serif"><br>
        7. Wash with 500 &micro;L - spin and discard flow through<br>
        8. Wash again with 500 &micro;L wash <br>
        9. spin on more time</font></p>
      <p><font face="Arial, Helvetica, sans-serif">10. take mini-column and place in microfuge tube add 50 &micro;L of elution buffer and let sit for 2 minutes.<br>
      11. centrifuge to collect DNA (plasmid).  </font></p>
      <p><font face="Arial, Helvetica, sans-serif">label your tube - plasmid all tubes in labelled plastic bag with plasmid name, date, your name and whether it has been confirmed by PCR or digest - plasmid can be stored frozen. .&nbsp;</font></p>
      <p><font face="Arial, Helvetica, sans-serif"><strong>Wizard (Promega) minipreps:</strong><br>
        <br>
        1. Pellet 1.5ml of bacteria from an overnighter (3 min. spin in minifuge).<br>
        2. Discard supernatants / resuspend in 200 &micro;Lresuspension buffer.<br>
        3. add 200 &micro;Lcell lysis buffer - invert until clear.<br>
        4. add 200 &micro;Lneutralization buffer - invert and then spin in microfuge for 
        5 minutes. <br>
        5. take supernatant - transfer to a new tube then add 0.8ml of resin <br>
        - invert and let sit for 1 min.<br>
        6. attach mini-column to 3ml syringe / or manifold (syringes can be reused!) 
        <br>
        and load with resin/cell supernatant.<br>
        7. push material through the mini-column<br>
        8. remove syringe, pull out plunger, replace syringe. add 2ml wash solution 
        - <br>
        push through the mini-column.<br>
        10. take mini-column and place in microfuge tube - centrifuge for 20 sec. 
        <br>
        11. take mini-column and place on fresh tube, add 50mL 65&deg;C water, centrifuge 
        again for 20 sec. -- your DNA is in the microfuge tube!<br>
      </font></p>
      <p><font face="Arial, Helvetica, sans-serif"><strong>Resuspension buffe</strong>r: 50mM Tris pH 7.5<br>
        10mM EDTA<br>
        100mg/ml RNAase A<br>
      </font></p>
      <p><font face="Arial, Helvetica, sans-serif"><strong>Lysis buffer</strong>: 0.2M NaOH<br>
        1% SDS<br>
      </font></p>
      <p><font face="Arial, Helvetica, sans-serif"><strong>Neutralization buffer</strong>: 1.32 M Kacetate pH 4.8<br>
      </font></p>
      <p><font face="Arial, Helvetica, sans-serif">Ethanol wash buffer: 0.2M NaCl<br>
        20mM Tris pH 7.5<br>
      5mM EDTA ADD 70ml 100% ethanol / 50ml wash buffe</font></p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="146"> <font size="-1" face="Arial, Helvetica, sans-serif"><em><strong><a href="../methods.htm" target="_self"><img src="../Graphics/ArrowLEFT.GIF" width="12" height="12" border="0"> Table of Methods</a></strong></em></font></td>
    <td width="448"><div align="right"><font size="-2" face="Arial, Helvetica, sans-serif"><a href="#top">top 
        </a></font></div></td>
  </tr>
</table>
<p><font color="#000000" face="Book Antiqua"><br>
  &nbsp; </font> 
</body>
</html>
